### PR TITLE
Update demo app and test: add second web page

### PR DESCRIPTION
**Pull request by Disi77**

### CODE REVIEW

File: demo_app.py [hunk 1]
```diff
@@ -12,6 +12,22 @@ def index():
...
+    </html>
+    """
+
+
+@app.route('/second_page')
+def second_page():
+    return """
+    <html>
+        <head>
+            <title>Hello world</title>
```
Comment (adelpopelkova) [inline demo_app.py:27]: The `title` is the same one as the one for the index page.

File: demo_app.py [hunk 2]
```diff
@@ -12,6 +12,22 @@ def index():
         </head>
         <body>
             Hello world!
+            <br>
+            <a href='/second_page'>LINK</a> to second page.
```
Comment (adelpopelkova) [inline demo_app.py:16]: Maybe change to `<a href='/second_page'>LINK</a> to the second page.`

**Comment by adelpopelkova** [comment]
Otherwise it looks good. If you'd apply the changes, you shouldn't forget to also change it in the tests.
But [this](https://github.com/encukou/freezeyt/issues/2#issuecomment-654865252) comment says, that it would be ideal, if the demo app for the test was separate. (Or that I don't understand the comment.)

**Comment by Disi77** [comment]
Because I want to do some changes

**Comment by adelpopelkova** [comment]
I am not 100 % sure, (because I never tried it), but you should be able to add commits to the PR, unless you are pushing them to a different branch.

**Comment by Disi77** [comment]
Yes I know ... but there was another issue with this job and it was easier to close this PR and do it again

**Pull request closed without merging**